.github/workflows/ci_tests.yml

```diff
@@ -58,13 +58,23 @@ jobs:
           python -c "import torch; assert torch.cuda.is_available()"
       - name: Test with pytest
         env:
-          # Configure endpoints
+          # Configure endpoints for Azure OpenAI
           AZUREAI_CHAT_ENDPOINT: ${{ secrets.AZUREAI_CHAT_ENDPOINT }}
```

Comment:
We should probably change the 'ENDPOINT' variables here to be `vars` rather than `secrets`